web-client/public/js/graph.js

```diff
@@ -1235,12 +1235,7 @@ export var drawGraph = function (workbook) {
     };
 
     const hasExpressionData = sheets => {
-        for (var property in sheets) {
-            if (property.match(ENDS_IN_EXPRESSION_REGEXP)) {
-                return true;
-            }
-        }
-        return false;
+        return Object.keys(sheets).some(property => property.match(ENDS_IN_EXPRESSION_REGEXP));
```

Comment:
Thanks, this looks good! But it looks like this function is duplicated in _update-app.js_? (see line 515 below) Should these functions be consolidated into one?

Sorry I didn’t notice this before…

Reply:
I didn't notice it either ....